npm/test/clientfileprocessing.test.ts

```diff
@@ -172,12 +172,12 @@ test("client file metadata successfully uploaded", async () => {
   ).resolves.not.toThrow()
 })
 
-test("progressCallback function updates the progress bar with the percentage processed", () => {
+test("metadataProgressBarCallback function updates the progress bar to a maximum of 50 percent", () => {
```

Comment:
The function name in the test description should match the actual function name. This is the same for all the new tests.